### PR TITLE
Fixed 

### PR DESCRIPTION
**Pull request by seunbayo**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -56,6 +56,11 @@ module.exports = {
       label: "Templates",
       items: ["templates/koii-template"],
     },
+    {
+      type: "category",
+      label: "tutorial",
```
Comment (rekpero) [inline sidebars.js:61]: update this as well

File: versioned_docs/version-3.0.0-beta/quick-start/minting-101.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+***Wait for the loader to load...***
+
+<img src={require('@site/static/img/3.0.0/loader.png').default} /> <br/><br/>
+
+### voila!! You have just minted your NFT
+
+<img src={require('@site/static/img/3.0.0/finalnft.png').default} /> <br/><br/>
+
+#### Go back to your App by clicking on Launch App.
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/quick-start/minting-101.md:56]: Elaborate this step a bit like, you need to sign a message that will give you access to the platform, and from next time, you don't have to go through this process of validating and minting the NFT anymore.

Another note here: Transfering the NFT to a different wallet will revoke the access to the platform, you need to hold the NFT if you want to use the platform.

File: versioned_docs/version-3.0.0-beta/quick-start/minting-101.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Final step
+When the tokens have been reflected  in your wallet, go back to your NFT minting Page and click on **MINT A KEY** as shown in [step3](#step-3-minting-an-nft). Confirm the transaction as seen below
+<img src={require('@site/static/img/3.0.0/finalb.png').default} /> <br/><br/>
+
+***Wait for the loader to load...***
+
+<img src={require('@site/static/img/3.0.0/loader.png').default} /> <br/><br/>
+
+### voila!! You have just minted your NFT
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/quick-start/minting-101.md:52]: put another note here: if you see an error popping in this step, please refresh the window and just click on find out again, you will be able to view your NFTs and launch the app

**Review by rekpero — CHANGES_REQUESTED** [review]
@seunbayo some more small comments

**Comment by seunbayo** [comment]
commit pushed

**Review by rekpero — APPROVED** [review]

**Pull request merged**